domain/domain.go

```diff
@@ -711,78 +712,86 @@ func (do *Domain) refreshMDLCheckTableInfo() {
 }
 
 func (do *Domain) mdlCheckLoop() {
-	ticker := time.Tick(time.Millisecond * 50)
+	var ticker <-chan time.Time
+	if intest.InTest {
+		ticker = time.Tick(time.Millisecond * 2)
+	} else {
+		ticker = time.Tick(time.Millisecond * 50)
```

Comment:
Should we do the same here? (Add a variable.)

RestLGTM

Reply:
Yes, sure.